kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -955,6 +955,20 @@ object KyuubiConf {
       .timeConf
       .createWithDefault(Duration.ofSeconds(120).toMillis)
 
+  val ENGINE_OPEN_MAX_ATTEMPTS: ConfigEntry[Int] =
+    buildConf("kyuubi.session.engine.open.max.attempts")
+      .doc("The number of times an open engine will retry when encountering a special error.")
+      .version("1.7.0")
+      .intConf
+      .createWithDefault(9)
+
+  val ENGINE_OPEN_INTERVAL: ConfigEntry[Long] =
+    buildConf("kyuubi.session.engine.open.interval")
+      .doc("The interval for opening engine.")
```

Comment:
kyuubi.session.engine.open.retry.wait

      .doc("How long to wait before retrying to open engine after a failure.")